app/controllers/projects_controller.rb

```diff
@@ -21,12 +21,18 @@ def create
     project_metadata.create(params: metadata_params)
     if new_project.save
       mailer = TigerdataMailer.with(project_id: @project.id)
-      mailer.project_creation.deliver_later
+      message_delivery = mailer.project_creation
+      message_delivery.deliver_later
 
       redirect_to project_confirmation_path(@project)
     else
       render :new
     end
+  rescue RedisClient::CannotConnectError => redis_connect_error
+    error_message = "Failed to connect to Redis: #{redis_connect_error}"
+    Rails.logger.error(error_message)
```

Comment:
@jrgriffiniii I like the idea of rescuing and logging these errors, but shouldn't we also notify honeybadger? I'm concerned that we never actually look at our rails error log unless we're trying to track down something specific, whereas we should be looking at honeybadger every day to find out if there are problems that need attention. If we had a serious problem with redis we'd likely not know without the honeybadger alerts. 

What happens to the `deliver_later` in this scenario? Does this message get silently dropped? That seems like a problem. Can we talk about what the user experience should be instead? Showing the user a redis connection error seems less than ideal: the user can't do anything with this information. Maybe instead something like "We experienced an error and could not deliver this email message?"